fab/core.py

```diff
@@ -68,7 +82,32 @@ def flow_reverse_kl(self, batch_size: int) -> torch.Tensor:
     def flow_alpha_2_div(self, batch_size: int) -> torch.Tensor:
         x, log_q = self.flow.sample_and_log_prob((batch_size,))
         log_p = self.target_distribution.log_prob(x)
-        return -torch.logsumexp(2 * (log_p - log_q), 0)
```

Comment:
@VincentStimper we want to minimise $p^2/q$ therefore I deleted this minus sign.

Reply:
Yes, you are right, thanks!